docs/installation-configuration.md

```diff
@@ -58,7 +56,7 @@ Example for using the `assemblyArtifact` parameter:
 </plugin>
 ```
 
-#### Using a repository
+#### Using a repository (Websphere Liberty Only)
```

Comment:
Do we need to add anything to the code to handle Open Liberty here (send warning message or error message)?

Reply:
A build exception is thrown in the Ant task when the runtime type is invalid. See https://github.com/WASdev/ci.ant/blob/master/src/main/java/net/wasdev/wlp/ant/install/WasDevInstaller.java#L142